omf/scripts/geoh5_to_omf.py

```diff
@@ -0,0 +1,19 @@
+import sys
+from pathlib import Path
+
+from omf.fileio import OMFWriter
+from omf.fileio.geoh5 import GeoH5Reader
+
+
+def run():
+    geoh5_filepath = Path(sys.argv[1])
+    if len(sys.argv) < 3:
+        output_filepath = geoh5_filepath.with_suffix(".omf")
+    else:
+        output_filepath = Path(sys.argv[2])
+    reader = GeoH5Reader(geoh5_filepath)
+    OMFWriter(reader(), output_filepath)
```

Comment:
Returning a `Path` object makes the current code to crash here. 

Traceback (most recent call last):
  File "C:\Users\dominiquef\Documents\GIT\mira\geoapps\assets\geoh5_to_omf.py", line 19, in <module>
    run()
  File "C:\Users\dominiquef\Documents\GIT\mira\geoapps\assets\geoh5_to_omf.py", line 15, in run
    OMFWriter(reader(), output_filepath)
  File "C:\Users\dominiquef\Documents\GIT\mira\omf\omf\fileio\fileio.py", line 44, in __init__
    if fname.endswith("geoh5"):
AttributeError: 'WindowsPath' object has no attribute 'endswith'


We need to either update fileio to handle it, or convert to str here.